pkg/api/v1/testkube/model_test_suite_execution_extended.go

```diff
@@ -243,3 +243,12 @@ func (e *TestSuiteExecution) EscapeDots() *TestSuiteExecution {
 func (e *TestSuiteExecution) UnscapeDots() *TestSuiteExecution {
 	return e.convertDots(utils.UnescapeDots)
 }
+
+func (e *TestSuiteExecution) CleanStepsOutput() *TestSuiteExecution {
+	for i := range e.StepResults {
+		if e.StepResults[i].Execution != nil && e.StepResults[i].Execution.ExecutionResult != nil {
+			e.StepResults[i].Execution.ExecutionResult.Output = ""
```

Comment:
might be we need to give a reference to test execution log?

Reply:
Added api spec note